src/ui/components/atoms/tooltips_II/TooltipsII.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from './TooltipsII.module.scss';
+import type { IProps } from './types/IProps';
+
+function Tooltip(props: IProps) {
+  const { children, position = 'bottom', text = 'default', size = 'medium' } = props;
+  return (
+    <div
+      className={`${styles.tooltip} ${styles[`tooltip-${position}`]} ${styles[`tooltip-${size}`]} `}
```

Comment:
use this approach to create your variants styles https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx#L7-L10